test/src/tests-persistence.ts

```diff
@@ -963,44 +997,48 @@ describe('migrations and persistence-related', function() {
             },
         };
 
-        var expires = new Date(
+        const expires = new Date(
             new Date().getTime() + 365 * 24 * 60 * 60 * 1000
-        ).toGMTString();
+        ).toString();
 
-        var cookiesWithExpiration = mParticle
+        const cookiesWithExpiration = mParticle
             .getInstance()
             ._Persistence.reduceAndEncodePersistence(
-                cookies,
+                // FIXME: We shouldn't have to force this
+                cookies as IPersistenceMinified,
                 expires,
                 'testDomain',
                 mParticle.config.maxCookieSize
             );
-        var cookiesWithoutExpiration = cookiesWithExpiration.slice(
+        const cookiesWithoutExpiration = cookiesWithExpiration.slice(
             0,
             cookiesWithExpiration.indexOf(';expires')
         );
-        var cookiesResult = JSON.parse(
+        const cookiesResult = JSON.parse(
             mParticle
                 .getInstance()
+                // FIXME: decodePersistence: string or object?
+                // @ts-ignore
                 ._Persistence.decodePersistence(cookiesWithoutExpiration)
         );
 
-        Should(cookiesResult['mpid1']).not.be.ok();
-        Should(cookiesResult['mpid2']).not.be.ok();
-        Should(cookiesResult['mpid3']).be.ok();
+        expect(cookiesResult['mpid1']).to.not.be.ok;
+        expect(cookiesResult['mpid2']).to.not.be.ok;
+        expect(cookiesResult['mpid3']).be.ok;
         cookiesResult.gs.csm[0].should.equal('mpid3');
         cookiesResult.should.have.property('mpid3');
-        mParticle.maxCookieSize = 3000;
+        // FIXME: Should this be in the config or not?
+        mParticle.config.maxCookieSize = 3000;
```

Comment:
The only thing I can thin of is this is done to revert the maxCookieSize back to the default size of 3000 since it is set to 400 on line 960.  Perhaps adding a comment here would have saved us this confusion, so I'd say add one.  Then again it gets set in the next few tests as well.  If tests pass if you remove this, then I'd say it's safe to remove.

Reply:
The placement of this assignment at the end of the test is weird to me, but there's also an inconstancy in the SDK interface. Technically `maxCookieSize` is not a property of `MParticleSDK`, but in the original version of this diff, it is. My question is whether or not it _should_ be.

Reply:
The placement of this assignment at the end of the test is weird to me, but there's also an inconstancy in the SDK interface. Technically `maxCookieSize` is not a property of `MParticleSDK`, but in the original version of this diff, it is. My question is whether or not it _should_ be.